### PR TITLE
EN: A Whole Bunch Of Pet UI

### PR DESCRIPTION
**Pull request by DabirSA**
Doesn't translate everything (haven't touched Limit Breaking for example), but does translate a lot of things people are commonly going to run into, as well as a few things that they probably won't. I've been good and made each change as a separate commit so it's easy to revert specific things for fixing.

### CODE REVIEW

File: UI/ui_pet.csv [hunk 1]
```diff
@@ -122,70 +122,70 @@ Warning_MaxAll_Title#0,"""Egg Synthesis"""
...
-ErrorServerBug#0,"""サーバーとの通信に失敗しました。<br><br>時間をおいて再度お試しください。"""
-ErrorClientBug#0,"""予期せぬ問題が発生しました。<br><br>時間をおいて再度お試しください。"""
-ErrorDBAccess#0,"""データベースのアクセスに失敗しました。<br><br>時間をおいて再度お試しください。"""
+ErrorServerBug#0,"""Failed to communicate with the server.<br><br>Please try again later."""
+ErrorClientBug#0,"""An unexpected error occurred.<br><br>Please try again later."""
+ErrorDBAccess#0,"""Database access failed.<br><br>Please try again later."""
 ErrorPetTypeInvalid#0,"""ペットの種類が正しくありません。"""
-ErrorPetHasFound#0,"""既にペットを所持しています。"""
-ErrorPetNotFound#0,"""ペットが見つかりませんでした。"""
+ErrorPetHasFound#0,"""Pet is already owned."""
```
Comment (SynthSy) [inline UI/ui_pet.csv]: Consider rewriting this, sounds really off. Something like "You already own this pet."

File: UI/ui_pet.csv [hunk 2]
```diff
@@ -122,70 +122,70 @@ Warning_MaxAll_Title#0,"""Egg Synthesis"""
...
+ErrorEggCanNotHatch#0,"""This egg cannot be hatched.<br>It can only be used in Egg Synthesis."""
+ErrorEggRarityUnknown#0,"""Could not determine egg rarity."""
+ErrorEggLock#0,"""Locked eggs<br>cannot be selected."""
 ErrorCandyTypeInvalid#0,"""キャンディーの種類が正しくありません。"""
-ErrorCandyNotFound#0,"""キャンディーが見つかりませんでした。"""
-ErrorCandyDuplicate#0,"""キャンディーのセットができませんでした。"""
-ErrorCandyOutOfBox#0,"""キャンディーのセットができませんでした。"""
-ErrorCandyNotArrangement#0,"""キャンディーのセットができませんでした。"""
-ErrorCandyTypeFull#0,"""キャンディーのセットができませんでした。"""
+ErrorCandyNotFound#0,"""Candy was not found."""
```
Comment (SynthSy) [inline UI/ui_pet.csv]: Rewrite this. Something like "Could not find any Candy"

File: UI/ui_pet.csv [hunk 3]
```diff
@@ -77,16 +77,16 @@ RemoverTitle#0,"""Remove Candy"""
...
+CompositeRemainTime#0,"""Remaining: $(0) minutes"""
+CompositeRemainGauge#0,"""+$(0)(+<pict pet_stone_red>$(1))"""
 CompositeRemainGaugeExtendPay#0,"""+$(0)"""
-CompositeRemainGaugeExtendNotPay#0,"""（+<pict pet_stone_red>$(0)）"""
-MaxCountError#0,"""この種類のキャンディーは<br>これ以上セットできません。"""
+CompositeRemainGaugeExtendNotPay#0,"""(+<pict pet_stone_red>$(0))"""
+MaxCountError#0,"""You cannot add any more<br>of this type of candy."""
 AttributeWarning#0,"""属性値が上限を超えてセットされたため、<br>上限値に合わせて、余剰分はカットされました。"""
-IceClushRarityError#0,"""実行できませんでした。"""
+IceClushRarityError#0,"""Caramel cube removal was not possible."""
```
Comment (SynthSy) [inline UI/ui_pet.csv]: "Could not remove the Caramel Cube."

**Review by SynthSy — CHANGES_REQUESTED** [review]

**Comment by DabirSA** [comment]
Changes made per SynthSy.

**Pull request merged**